README.md

```diff
@@ -1 +1,96 @@
 # Wiki
+## Don't use deprecated tags
+In html many tags have been deprecated. That means they are to be phased out. While some tags are deprecated because they really don't serve a purpose anymore, most are deprecated in favor of style sheets, which can control visual effects such as alignment, spacing, and more.
+
+### Example
+Various html tags that are deprecated are 
+1. 	acronym
+2. 	applet
+3. 	big
+4.	blink
+5. 	basefont
+6. 	center
+7. 	dir
+8. 	frame
+9. 	frameset
+10. isindex
+11. listing
+12. noembed
+13. plaintext
+14. spacer
+15. strike
+16. tt
+17. xmp
+
+#### Not Recommended - Recommended
+
+
+#####font, basefont 
+These tags should not be used any more. CSS is preferred to serve the same purpose.
+
+
+#####b, i, u
+These tend to be a bit more controversial, but as much as possible, try to use respectively strong, em or span tag and CSS (text-decoration:underline) when relevant. 
+
+
+#####acronym
+The acronym tag should not be used any more. abbr is preferred to represent an abbreviation.
+
+
+#####tt, xmp
+Do not use these elements. To represent a fragment of computer code, use code tag. To represent preformatted text, use pre tag. If you only want to use a monospaced font, you can use font-family: monospace in CSS.
+
+
+#####applet
+This element shouldn't be used any more. Instead object tag is preferred because it is more generic.
+
+
+#####center
+This tag has been deprecated in favour of text-align property of CSS which can be applied to the div element or to an individual p element.
+
+
+#####spacer
+spacer tag is an HTML element which is used for inserting white spaces to web pages. However spacer tag is not supported by any major browser and same effects can be created with various CSS rules.
+
+
+#####blink
+This tag should not be used and a replacement for it is {text-decoration: blink} which can be applied to the div element or to an individual p element.
+
+
+#####dir
+dir tag has been deprecated in favor of ul tag because there is no need for two tags that do the same job.
+
+
+#####strike
+The HTML Strikethrough Element renders text with a strikethrough. However this tag is deprecated and replaced with {text-decoration: line-through}.
+
+
+#####frame, frameset
+It is not recommended to use frame tag which defines a particular area in which another HTML document can be displayed and frameset tag which is used to contain frame elements.
+
+	<frameset cols="20%,*,">
```

Comment:
make this a HTML code segment, see https://help.github.com/articles/github-flavored-markdown/#syntax-highlighting